Console/UpgradeHelperCommand.php

```diff
@@ -97,9 +97,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         foreach ($report as $type => $items) {
             $output->writeln('-------- ' . $type . ' --------');
-            foreach ($items as $patched => $customized) {
-                $output->writeln('Patched: ' . $patched);
-                $output->writeln('Customized: ' . $customized);
+            if (count($items) > 0) {
```

Comment:
I would invert this to reduce intentation...

```
if (count($items) === 0) {
    $output->writeln('No customizations found for ' . $type);
    continue;
}

// Rest of the code runs here
```

Reply:
Also, a nit, but can we `strtolower()` `$type`

Reply:
Done!